source/dom.js

```diff
@@ -1,4 +1,10 @@
-const div = document.createElement('div');
+import { isBrowserEnv } from './utils';
+
+const fakeWindow = {};
+
+export const global = isBrowserEnv ? window : fakeWindow;
+
+const div = isBrowserEnv ? document.createElement('div') : {};
```

Comment:
Yeah, but @effrenus will be the one who changed this and git blame will point him for a while :) I think it's better to refactor some places as we run into them, it's convenient time to do that